appliances/Harbor/appliance.sh

```diff
@@ -210,6 +211,32 @@ download_unpack_harbor() {
 }
 
 
+mount_persistent() {
+    # Check if /data directory exists, if not create it
+    if [ ! -d "/data" ]; then
+        mkdir -p /data
+    fi
+
+    # Check if /dev/vda exists
+    if [ -e "/dev/vda" ]; then
```

Comment:
We'll probably need a better way of getting the device, maybe a USER INPUT? so vda will depend on the driver used (e.g. sd vs vd) and the number of disks in the VM template. So probably the safest thing is to let the user input which device should be used for persistent data